stories/multi-combo-box/multi-combo-box.stories.js

```diff
@@ -0,0 +1,960 @@
+                        <span class="fd-checkbox__text fd-list__title">
+                            Lemon
+                        </span>
+                    </label>
+                </li>
+            </ul>
+        </div>
+        </div>
+    </div>
+</div>
+`;
+
+matchPopoverBodySize.parameters = {
+    docs: {
+        iframeHeight: 300,
+        storyDescription: `
+The default length size of the popover body is often different from the text length. 
+The body length can be adjusted to match the text length by adding the \`fd-popover__body--dropdown-fill\` class to the \`fd-popover__body\`.
+
+This class has been added to all the \`Multi ComboBox\` examples above. 
+In the example you can see how the \`Multi ComboBox\` component looks without the \`fd-popover__body--dropdown-fill\` modifier.
+`
+    }
+};
+
+export const readOnlyAndDisabled = () => `<div style="display:flex;height:80px;">
+    <div class="fd-popover" style="display:flex;flex-direction:column;padding-right:1rem">
+        <div class="fd-popover__control" aria-controls="F4GcX3X8a" aria-expanded="false" aria-haspopup="false">
+            <div class="fd-input-group fd-input-group--control" readonly aria-readonly="true">
+                <div class="fd-tokenizer fd-tokenizer--readonly">
+                    <div class="fd-tokenizer__inner">
+                        <span class="fd-token fd-token--readonly" role="button">
+                            <span class="fd-token__text">
+                                Bibendum
+                            </span>
+                        </span>
+                        <span class="fd-token fd-token--readonly" role="button">
+                            <span class="fd-token__text">
+                                Lorem
+                            </span>
+                        </span>
+                        <span class="fd-token fd-token--readonly" role="button">
+                            <span class="fd-token__text">
+                                Dolor
+                            </span>
+                        </span>
+                        <span class="fd-token fd-token--readonly" role="button">
+                            <span class="fd-token__text">
+                                Filter
+                            </span>
+                        </span>
+                        <span class="fd-token fd-token--readonly">
+                            <span class="fd-token__text">
+                                Lorem
+                            </span>
+                        </span>
+                        <span class="fd-tokenizer__indicator">
+                            <div class="fd-popover">
+                                <div class="fd-popover__control fd-input-group__control" aria-controls="F4GF5348a" aria-expanded="false" aria-haspopup="true">
+                                    <a href="#" class="fd-link">
```

Comment:
This shouldn't be a link as it's not navigating to a different page, just opening a popover.